### DISCUSSION TITLE
Add Screenshot Feature

### DISCUSSION THREAD

**Discussion by Kinasho** [xgi/houdoku#139] [Feature Requests]
I would like for there to be a screenshot feature, where I can save images of manga to a specified location. 

For single page spreads and long strip manga, the image taken can be of the page that it is currently on (With regards to the long strip manga, only the page that it is on, not what the screen sees which might be multiple pages at once) and for double page manga, it would take an image of the two pages, preferably as one image file. 

There can be the addition of an option to choose the location of the screenshots folder, as well as the file type (jpg or png).

Also, if this screenshot feature were to be implemented, a keyboard shortcut such as "s" or "ctrl+s" can be used as the default keybind.

**Comment by xgi** [comment]
Good suggestion! Not sure I would prioritize this in the near future, though.
